app/models/vote.rb

```diff
@@ -13,26 +13,45 @@ class Vote < ActiveRecord::Base
   include Translatable
   is_translatable on: :statement
 
-  scope :for_user, lambda {|user_id| where(:user_id => user_id)}
-  scope :by_discussion, ->(discussion_id = nil) { joins(:motion).where("motions.discussion_id = ? OR ? IS NULL", discussion_id, discussion_id) }
+  scope :for_user, lambda {|user_id| where(user_id: user_id)}
+  scope :by_discussion, -> (discussion_id = nil) { joins(:motion).where("motions.discussion_id = ? OR ? IS NULL", discussion_id, discussion_id) }
   scope :most_recent, -> { where age: 0  }
 
-  delegate :name, :to => :user, :prefix => :user # deprecated
-  delegate :name, :to => :user, :prefix => :author
-  delegate :group, :discussion, :to => :motion
-  delegate :users, :to => :group, :prefix => :group
-  delegate :author, :to => :motion, :prefix => :motion
-  delegate :author, :to => :discussion, :prefix => :discussion
-  delegate :name, :to => :motion, :prefix => :motion
-  delegate :name, :full_name, :to => :group, :prefix => :group
-  delegate :locale, :to => :user
+  delegate :name, to: :user, prefix: :user # deprecated
```

Comment:
Hey I'd be happy to start enforcing this standard more for open source contributions (I personally vastly prefer is over the rocket). What do you think?